expanderr.go

```diff
@@ -617,8 +658,9 @@ func logic(w io.Writer, buildctx *build.Context, posn string) error {
 		// Hack: inline comments (e.g. os.Remove("/foo" /* path */)) get lost
 		// when formatting node, so we string-replace the formatted CallExpr
 		// with the original CallExpr.
+
 		var stmtFmt, ceFmt bytes.Buffer
-		if err := format.Node(&stmtFmt, e.fset, node); err != nil {
+		if err := format.Node(&stmtFmt, token.NewFileSet(), node); err != nil {
```

Comment:
Not sure why, but only with longer `no-err-callback`'s, This statement will format the node very strangely over multiple lines when using the same `e.fset` that is used everywhere else, and for some reason using a new one here fixes it. Have not seen any side affects from doing it this way, but it is not clear to me what its used for in this statement.

Reply:
Not entirely obvious to me either.

Ideally, we should change the code to use https://github.com/dave/dst to resolve the comment above regarding inline comments, and perhaps that would also address whichever weirdness we’re running into here :)